src/main.cpp

```diff
@@ -291,17 +292,20 @@ void getConfigSchemaJson(JsonVariant json)
     controller["type"] = "integer";
     controller["minimum"] = 1;
     controller["maximum"] = PWM_CONTROLLER_COUNT;
+    controller["description"] = "Specifies the PCA controller being declared - where the strip will be attached";
   }
   
   JsonObject strip = channelProperties.createNestedObject("strip");
   strip["type"] = "integer";
   strip["minimum"] = 1;
   strip["maximum"] = PWM_CHANNEL_COUNT;
+  strip["description"] = "Assigns a number designator to a strip - in order they are wired.";
```

Comment:
`Assigns an index to the strip, incrementing from 1, in the order it is wired to the controller`